### PR TITLE
Rerun polymer-modulizer on the latest code from master. Clean up some…

### PR DESCRIPTION
**Pull request by arobins**
… comments.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -24,10 +24,9 @@
...
   "dependencies": {
     "@polymer/polymer": "^3.0.0",
-    "@polymer/iron-jsonp-library": "^3.0.0-pre.19"
+    "@polymer/iron-jsonp-library": "^3.0.0-pre.18"
   },
   "devDependencies": {
-    "@polymer/iron-component-page": "^3.0.0-pre.19",
-    "@webcomponents/webcomponentsjs": "^2.0.0"
+    "@polymer/iron-component-page": "^3.0.0-pre.18"
   }
```
Comment (rictic) [inline package.json]: Are the changes to package.json intentional?

Comment (rictic) [reply package.json]: Hm, odd. Recommend reverting, it's nice for this to have a dev dep on webcomponentsjs

**Comment by rictic** [comment]
@e111077 mind giving this a quick review? I don't have owners of this repo

File: package.json (hunk 1 above)
Comment (arobins) [reply package.json]: Done.

**Pull request merged**